app/lib/methods/sendMessage.js

```diff
@@ -5,8 +5,9 @@ import reduxStore from '../createStore';
 import log from '../../utils/log';
 import random from '../../utils/random';
 
-export const getMessage = (rid, msg = '', tmid) => {
+export const getMessage = (rid, msg = '', tmid, shareExtension) => {
```

Comment:
This is not scalable.
You can send `userId` and `username` as params instead.

Reply:
Fixed.